amethyst_audio/src/systems/audio.rs

```diff
@@ -120,15 +80,22 @@ impl<'a> System<'a> for AudioSystem {
                         }
                     }
                     while let Some(source) = audio_emitter.sound_queue.pop() {
-                        let sink = SpatialSink::new(
-                            &listener.output.endpoint,
-                            emitter_position,
-                            left_ear_position,
-                            right_ear_position,
-                        );
-                        let atomic_bool = Arc::new(AtomicBool::new(false));
-                        sink.append(EndSignalSource::new(source, atomic_bool.clone()));
-                        audio_emitter.sinks.push((sink, atomic_bool));
+                        if SOUNDS_PLAYING.load(Ordering::Relaxed) < MAX_SOUNDS_PLAYING {
```

Comment:
Ordering::Relaxed
You also use it in output.rs.
Wouldn't it be better to have it configurable with a way to ensure that they are both the same value, in case someone changes the Ordering in only one place?